src/components/Counter/index.tsx

```diff
@@ -3,23 +3,31 @@ import Button from "../Button";
 import { CounterStyle, ButtonContainer } from "./styles";
 
 class Counter extends React.Component {
+  interval: NodeJS.Timeout | null = null;
+  audioRef: React.RefObject<HTMLAudioElement> = React.createRef();
+
+  url = require("../../Assets/bell.wav");
+
   state = {
     minutes: 0,
     seconds: 5,
-    isBreakOver: true,
+    isBreak: true,
   };
-  // usei esses valores (5 segundos) so pra enxergar mais rapido a mudança. task eh 5 segundos e pausa eh 4 segundos
+  // usei esses valores (5 segundos) so pra enxergar mais rapido a mudança.
 
   Countdown = () => {
-    const counterLogic = () => {
+    this.interval = setInterval(() => {
       if (this.state.seconds === 0 && this.state.minutes === 0) {
-        clearInterval(interval);
+        this.clearInterval();
         this.setState({
-          minutes: 0,
-          seconds: this.state.isBreakOver ? 4 : 5,
-          isBreakOver: !this.state.isBreakOver,
+          minutes: this.state.isBreak ? 5 : 25,
+          isBreak: !this.state.isBreak,
+          seconds: 0,
         });
-        return 
+        if (this.audioRef.current) {
```

Comment:
[suggestion] You could remove the if and use optional chaining here instead, like so 
```suggestion
        this.audio?.current?.play()
```

Reply:
Thanks Gustavo!!! Just changed.